k8s/utils/health/cluster_health_check.py

```diff
@@ -46,7 +46,7 @@ def checkCluster(node_count):
     while True:
         try:
             count = get_node_status(node_count)
-            if count == int(node_count) + 1: # +1 master node
+            if count == int(node_count):
```

Comment:
IMO, should be good to get a fairly accurate count of nodes/state of cluster. The client APIs going through - itself in a way is indicative of availability of *master* (in both cases where master(s) are present or *masterless* clusters like GKE )